src/plugins/core/header_visibility.ts

```diff
@@ -27,9 +27,14 @@ export class HeaderVisibilityPlugin extends CorePlugin {
           cmd.dimension === "COL"
             ? this.getters.getNumberCols(cmd.sheetId)
             : this.getters.getNumberRows(cmd.sheetId);
-        return (hiddenGroup || []).flat().concat(cmd.elements).length < elements
-          ? CommandResult.Success
-          : CommandResult.TooManyHiddenElements;
+        const hiddenElements = (hiddenGroup || []).flat();
+        if (hiddenElements.length >= elements) {
+          return CommandResult.TooManyHiddenElements;
```

Comment:
Idk if we really want to support it, but techincally we could do dispatch("HIDE_COL", indexes : [ 0,0,0,0,0,0,0,0,0,0,0,0,0....] and then this condition would be wrong. That's a bit of an edge casez and a malformed command tho.

Reply:
Wait no, you meant to compare `cmd.elements.length  + hiddenElements.length>= elements` right ? You compare the elements currently hidden in the sheet, with the number of elements in the sheet there.

Reply:
yup I messed up the adaptation -  I fixed the condition and cast everything in sets to purge multiple entries as well  -  I feel like we should have the command purge at the start of dispatch inside model but we have to discuss this. model does not know what goes on in the plugins but it sure does know some of the commands - we could argue about their definition ... @LucasLefevre

Reply:
I'm not sure to understand what you mean